sqldelight-gradle-plugin/src/main/kotlin/com/squareup/sqldelight/gradle/SqlDelightExtension.kt

```diff
@@ -50,6 +50,28 @@ open class SqlDelightExtension {
     return Unit
   }
 
+  /**
+   * Supports configuration in Kotlin script build files.
+   *
+   * sqldelight {
+   *   "MyDatabase" {
```

Comment:
Can we just expose a normal function? I'd strongly prefer
```kotlin
database("MyDatabase") {
  // ..
}
```
instead of trying to be cute and make it look like Groovy. We don't want to fall into the same trap as the Kotlin Gradle DSL did.

Reply:
oooo. thats nice

Reply:
Works for me. I was just following the `kotlin-dsl` lead for things like dependency configurations and didn't realize it was a trap. I thought the goal of that was to keep both config mechanisms as close as possible.

I like that regular method dsl even better.